app/models/identity.rb

```diff
@@ -9,8 +9,10 @@ def self.find_with_omniauth(auth)
   def self.create_with_omniauth(auth)
     Rails.logger.debug('Creating user from')
     Rails.logger.debug(auth)
-    user = User.create_with_omniauth(auth.info, auth.provider)
-    Rails.logger.debug(user.inspect)
-    create(uid: auth['uid'], provider: auth['provider'], user_id: user.id)
+    transaction do
+      user = User.create_with_omniauth(auth.info, auth.provider)
+      Rails.logger.debug(user.inspect)
+      create!(uid: auth['uid'], provider: auth['provider'], user_id: user.id)
```

Comment:
`User.create_with_omniauth` uses the bang version (`create!`) to raise any errors, so I thought it made sense that this method did the same.

https://github.com/nbgallery/nbgallery/blob/cd6116dd11cf417d17d900aaf511a524d9329260/app/models/user.rb#L576-L586